Playbooks/AS-Slack-Integration/azuredeploy.json

```diff
@@ -0,0 +1,505 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "comments": "This playbook is intended to be run from a Microsoft Sentinel Incident. It will generate a post in a Slack channel you specify with the Microsoft Sentinel Incident and entity information.",
+        "author": "Accelerynt"
+    },
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "AS-Slack-Integration",
+            "type": "String"
+        },
+        "OAuthToken": {
+            "defaultValue": "Enter your Slack Oauth Token, starting with xoxb-",
+            "type": "String"

```

Comment:
The type should be **securestring** as this is a secret.

Reply:
This is resolved